src/app/components/ScrollablePromo/PromoList/index.jsx

```diff
@@ -12,12 +12,13 @@ import useOperaMiniDetection from '#hooks/useOperaMiniDetection';
 import Promo from '../Promo';
 
 const ScrollPromo = styled.ul`
-  display: flex;
   list-style: none;
   ${({ dir }) => `padding-${dir === 'ltr' ? 'left' : 'right'}: 0;`}
   margin: 0;
-  ${({ isOperaMini }) => (isOperaMini ? `flex-direction: column;` : '')}
-
+  ${({ isOperaMini }) =>
+    isOperaMini
+      ? ``
+      : `display: flex;
```

Comment:
Won't having `display:flex` cause problems in IE11 as it's only partially supported according to this https://caniuse.com/?search=display%3Aflex (we do still support IE11, right?)

Reply:
@aeroplanejane  > IE11 is end of life in July - not a browser we’re actively supporting anymore